inst/schema/report_run_status_response.json

```diff
@@ -0,0 +1,35 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "properties": {
+        "time_queued": {
+             "type": "number",
+             "description": "time queued in UTC time."
+        },
+        "time_started": {
+             "type": ["number", "null"],
+             "description": "time started run in UTC time."
+        },
+        "time_complete": {
+             "type": ["number", "null"],
+             "description": "time completed in UTC time."
+        },
+        "queue_position": {
+          "type": ["number", "null"]
+        },
+        "logs": {
+             "type": ["array", "null"],
+             "items": {
+               "type": "string"
+             }
+        },
+        "status": {
+          "type": "string"
```

Comment:
Might be nice to specify the list of allowed values here. Or would that be too restrictive?